graphql/schema.graphql

```diff
@@ -321,5 +322,20 @@ type LogMessage {
   version: Int
 }
 
+type CommitQueue {
+  ProjectID: String
+  Queue: [CommitQueueItem]
```

Comment:
The elements in the rest model can't be null, so this should be CommitQueueItem!